packages/core/src/components/header/header.tsx

```diff
@@ -305,6 +311,19 @@ export class Header {
                           )}
                         </li>
                       )}
+                      <li>
+                        {this.helpUrl ? (
+                          <a href={this.helpUrl} onClick={(e) => this.clickHandler(e, "help", { url: this.helpUrl })}>
```

Comment:
Ik begrijp dat het in de rest van de file ook niet is gedaan, maar wat ik nog niet helemaal begrijp:

Waarom heeft deze `<a>` geen `dso-tertiary` ? Visueel moet het er toch altijd als een tertiary button uitzien, ongeacht `<button>` of `<a>` ?

Reply:
Dat klopt, echter bij een button moeten allerlei soorten styling eerst ongedaan worden gemaakt en bij de link was het stylen ervan een stuk eenvoudiger. Ik denk dat dat de reden is. Nu zie ik wel een verschil als ik de class op de link gebruik, omdat er dan gratis een stukje opmaak meekomt vanuit het icoontje (een `margin-left: 8px`). Ik ga het gelijktrekken (ook met de andere links/buttons)

Reply:
<img width="460" alt="Scherm­afbeelding 2023-10-24 om 11 29 21" src="https://github.com/dso-toolkit/dso-toolkit/assets/7202272/95dd62c4-51c3-43cb-97b2-06876d78d1c8">

Bijzonder, er zit een hoogte/afstandsverschil tussen de onderlijning en de linktekst (ik dacht eerst dat het door het icoontje rechts kwam, maar dat blijkt het niet te zijn).

Reply:
Aha. Goed gezien.

Het component "Button: Tertiary" zou in principe op `<button>` en `<a>` kunnen, visueel zouden ze identiek moeten zijn.

Fijn dat je het gelijk gaat trekken!

Reply:
Michiel is tevreden! Volgens mij kan deze door (naar PR review)

Reply:
👏